### DISCUSSION TITLE
如何上传本服务器文件

### DISCUSSION THREAD

**Discussion by hyy34** [AlistGo/alist#5082] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
我已经阅读了[文档](https://alist.nn.ci)。

- [X] I'm sure there are no duplicate issues or discussions.
我确定没有重复的issue或讨论。

- [X] I'm sure it's due to `AList` and not something else(such as [Network](https://alist.nn.ci/faq/howto.html#tls-handshake-timeout-read-connection-reset-by-peer-dns-lookup-failed-connect-connection-refused-client-timeout-exceeded-while-awaiting-headers-no-such-host) ,`Dependencies` or `Operational`).
我确定是`AList`的问题，而不是其他原因（例如[网络](https://alist.nn.ci/zh/faq/howto.html#tls-handshake-timeout-read-connection-reset-by-peer-dns-lookup-failed-connect-connection-refused-client-timeout-exceeded-while-awaiting-headers-no-such-host)，`依赖`或`操作`）。

- [X] I'm sure this issue is not fixed in the latest version.
我确定这个问题在最新版本中没有被修复。

### AList Version / AList 版本

v3.26.0

### Driver used / 使用的存储驱动

阿里云

### Describe the bug / 问题描述

服务器docker部署alist，想把服务器里的/www/test   文件夹的文件上传至阿里云，如何操作呢？看了下说明，没找到，只找到了用aria2下载，然后离线上传的说明。

### Reproduction / 复现链接

.

### Config / 配置

.

**Comment by xhofe** [comment]
将服务器的目录也作为一个目录挂载 然后复制到阿里云盘的文件夹

**Reply by hyy34** [reply]
使用alist v3吗？请问docker如何挂载服务器的目录呢？搜索了很多没找呢。

**Reply by xhofe** [reply]
https://docs.docker.com/storage/volumes/

**Reply by hyy34** [reply]
额，英文看不懂啊，就是docker的时候映射和本地文件夹被？比如 -v /test1:/test2，然后挂载test2的文件夹？

**Reply by hyy34** [reply]
是docker映射完文件夹，然后使用本地储存的方式挂载，然后从前端复制吗？您说的复制到阿里云盘的文件夹有点没看懂。。
